wpcom-thumbnail-editor.php

```diff
@@ -651,24 +705,23 @@ public function get_coordinates( $attachment_id, $size ) {
 	/**
 	 * Saves the coordinates for a custom crop for a given attachment ID and thumbnail size.
 	 *
-	 * @param $attachment_id int Attachment ID.
-	 * @param $size string Thumbnail size name.
-	 * @param $coordinates array Array of coordinates in the format array( x1, y1, x2, y2 )
+	 * @param int    $attachment_id Attachment ID.
+	 * @param string $size Thumbnail size name.
+	 * @param array  $coordinates Array of coordinates in the format array( x1, y1, x2, y2 ).
```

Comment:
May not have been flagged by the PHPCS, but the WPCS says to align the start of the descriptions as well (and any wrapped lines):

```suggestion
	 * @param int    $attachment_id Attachment ID.
	 * @param string $size          Thumbnail size name.
	 * @param array  $coordinates   Array of coordinates in the format `array( x1, y1, x2, y2 )`.
```